common/src/main/resources/less/gpp-grid.less

```diff
@@ -0,0 +1,91 @@
+@two-column-break: 60em;
+@three-column-break: 90em;
```

Comment:
We can play with these column breaks, but they will of necessity be inexact until CSS provides a way of querying container size, not just the viewport size.